code/modules/admin/verbs/highlander_datum.dm

```diff
@@ -0,0 +1,99 @@
+
+GLOBAL_DATUM(highlander_controller, /datum/highlander_controller)
+
+
+/**
+ * The highlander controller handles the admin highlander mode, if enabled.
+ * It is first created when "there can only be one" triggers it, and it can be referenced from GLOB.highlander_controller
+ */
+/datum/highlander_controller
+
+/datum/highlander_controller/New()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_CREWMEMBER_JOINED, .proc/new_highlander)
+	sound_to_playing_players('sound/misc/highlander.ogg')
+	send_to_playing_players("<span class='boldannounce'><font size=6>THERE CAN BE ONLY ONE</font></span>")
+	for(var/obj/item/disk/nuclear/N in GLOB.poi_list)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names